easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.5.3-foss-2021a-CUDA-11.3.1.eb

```diff
@@ -0,0 +1,244 @@
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+toolchainopts = {'pic': True}
+
+builddependencies = [
+    ('Bazel', '3.7.2'),
+    ('protobuf', '3.17.3'),
+    # git 2.x required, see also https://github.com/tensorflow/tensorflow/issues/29053
+    ('git', '2.32.0', '-nodocs'),
+    ('pybind11', '2.6.2'),
+    ('UnZip', '6.0'),
+]
+dependencies = [
+    ('CUDA', '11.3.1', '', True),
+    ('cuDNN', '8.2.1.32', versionsuffix, True),
+    ('NCCL', '2.10.3', versionsuffix),
+    ('Python', '3.9.5'),
+    ('h5py', '3.2.1'),
+    ('cURL', '7.76.0'),
+    ('double-conversion', '3.1.5'),
+    ('flatbuffers', '2.0.0'),
+    ('giflib', '5.2.1'),
+    ('hwloc', '2.4.1'),
+    ('ICU', '69.1'),
+    ('JsonCpp', '1.9.4'),
+    ('libjpeg-turbo', '2.0.6'),
+    ('LMDB', '0.9.28'),
+    ('NASM', '2.15.05'),
+    ('nsync', '1.24.0'),
+    ('SQLite', '3.35.4'),
+    ('PCRE', '8.44'),
+    ('protobuf-python', '3.17.3'),
+    ('flatbuffers-python', '2.0'),
+    ('typing-extensions', '3.10.0.0'),
+    ('libpng', '1.6.37'),
+    ('snappy', '1.1.8'),
+    ('zlib', '1.2.11'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+# Dependencies created and updated using findPythonDeps.sh:
+# https://gist.github.com/Flamefire/49426e502cd8983757bd01a08a10ae0d
+exts_list = [
+    ('Markdown', '3.3.4', {
+        'checksums': ['31b5b491868dcc87d6c24b7e3d19a0d730d59d3e46f4eea6430a321bed387a49'],
+    }),
+    ('pyasn1-modules', '0.2.8', {
+        'checksums': ['905f84c712230b2c592c19470d3ca8d552de726050d1d1716282a1f6146be65e'],
+    }),
+    ('rsa', '4.8', {
+        'checksums': ['5c6bd9dc7a543b7fe4304a631f8a8a3b674e2bbfc49c2ae96200cdbe55df6b17'],
+    }),
+    ('cachetools', '4.2.4', {
+        'checksums': ['89ea6f1b638d5a73a4f9226be57ac5e4f399d22770b92355f92dcb0f7f001693'],
+    }),
+    ('google-auth', '1.35.0', {
+        'modulename': 'google.auth',
+        'checksums': ['b7033be9028c188ee30200b204ea00ed82ea1162e8ac1df4aa6ded19a191d88e'],
+    }),
+    ('oauthlib', '3.2.0', {
+        'checksums': ['23a8208d75b902797ea29fd31fa80a15ed9dc2c6c16fe73f5d346f83f6fa27a2'],
+    }),
+    ('requests-oauthlib', '1.3.1', {
+        'checksums': ['75beac4a47881eeb94d5ea5d6ad31ef88856affe2332b9aafb52c6452ccf0d7a'],
+    }),
+    ('google-auth-oauthlib', '0.4.6', {
+        'checksums': ['a90a072f6993f2c327067bf65270046384cda5a8ecb20b94ea9a687f1f233a7a'],
+    }),
+    ('Werkzeug', '2.0.2', {
+        'checksums': ['aa2bb6fc8dee8d6c504c0ac1e7f5f7dc5810a9903e793b6f715a9f015bdadb9a'],
+    }),
+    ('absl-py', '0.15.0', {
+        'modulename': 'absl',
+        'checksums': ['72d782fbeafba66ba3e525d46bccac949b9a174dbf66233e50ece09ee688dc81'],
+    }),
+    ('astunparse', '1.6.3', {
+        'checksums': ['5ad93a8456f0d084c3456d059fd9a92cce667963232cbf763eac3bc5b7940872'],
+    }),
+    ('grpcio', '1.43.0', {
+        'modulename': 'grpc',
+        'checksums': ['735d9a437c262ab039d02defddcb9f8f545d7009ae61c0114e19dda3843febe5'],
```

Comment:
Previous EasyConfigs set a `preinstallopts: "export GRPC_PYTHON_BUILD_EXT_COMPILER_JOBS=%(parallel)s"`. Any idea if it's still relevant here? (see [here](https://github.com/easybuilders/easybuild-easyconfigs/blob/50885ea5e4ee9c44720c92b47fc14a17fa05d4c5/easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.5.0-fosscuda-2020b.eb#L91))

Again, if so, it would also apply to the non-cuda version of this EasyConfig, of course :)

Reply:
indeed, I missed that change. Thanks, fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/15019/commits/0a4ec23a5c4bf29042654f34abc670d5d2d3bcd9